crates/bevy_text/src/lib.rs

```diff
@@ -34,13 +34,18 @@ use bevy_render::{camera::CameraUpdateSystem, ExtractSchedule, RenderApp};
 use bevy_sprite::SpriteSystem;
 use std::num::NonZeroUsize;
 
+/// Adds text rendering support to an app.
+///
+/// When the `bevy_text` feature is enabled with the `bevy` crate, this
+/// plugin is included by default in the `DefaultPlugins`.
 #[derive(Default)]
 pub struct TextPlugin;
 
-/// [`TextPlugin`] settings
+/// Settings used to configure the [`TextPlugin`].
 #[derive(Resource)]
 pub struct TextSettings {
-    /// Maximum number of font atlases supported in a [`FontAtlasSet`].
+    /// Maximum number of font atlases supported in a [`FontAtlasSet`]. When this is exceeded,
```

Comment:
```suggestion
    /// Soft maximum number of font atlases supported in a [`FontAtlasSet`]. When this is exceeded,
```

Reply:
Soft maximum is a good way of describing it. We should rename the field as well, `soft_max_font_atlases` would be okay.